build/Build.cs

```diff
@@ -0,0 +1,440 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Nuke.Common;
+using Nuke.Common.CI.TeamCity;
+using Nuke.Common.IO;
+using Nuke.Common.ProjectModel;
+using Nuke.Common.Tooling;
+using Nuke.Common.Tools.DotNet;
+using Nuke.Common.Tools.GitVersion;
+using Nuke.Common.Tools.NuGet;
+using Nuke.Common.Utilities.Collections;
+using Serilog;
+using static Nuke.Common.IO.FileSystemTasks;
+using static Nuke.Common.Tools.DotNet.DotNetTasks;
+using static Nuke.Common.Tools.Git.GitTasks;
+
+namespace Calamari.Build
+{
+    class Build : NukeBuild
+    {
+        const string RootProjectName = "Calamari";
+
+        [Parameter("Configuration to build - Default is 'Debug' (local) or 'Release' (server)")]
+        readonly Configuration Configuration =
+            IsLocalBuild ? Configuration.Debug : Configuration.Release;
+
+        [Solution] 
+        [Required] 
+        readonly Solution? Solution;
+
+        [Parameter("Run packing step in parallel")] 
+        readonly bool PackInParallel;
+
+        [Parameter("Build verbosity")] 
+        readonly string BuildVerbosity = "Minimal";
+
+        [Parameter("Sign Binaries")] 
+        readonly bool SignBinaries;
+        
+        // When building locally signing isn't really necessary and it could take
+        // up to 3-4 minutes to sign all the binaries as we build for many, many
+        // different runtimes so disabling it locally means quicker turn around
+        // when doing local development.
+        bool WillSignBinaries => !IsLocalBuild || SignBinaries;
+
+        [Parameter("Append Timestamp")] 
+        readonly bool AppendTimestamp;
+
+        [Parameter("Set Calamari Version on OctopusServer")] 
+        readonly bool SetOctopusServerVersion;
+
+        [Parameter("AzureKeyVaultUrl")] 
+        readonly string? AzureKeyVaultUrl;
+
+        [Parameter("AzureKeyVaultAppId")] 
+        readonly string? AzureKeyVaultAppId;
+
+        [Parameter("AzureKeyVaultAppSecret")]
+        [Secret]
+        readonly string? AzureKeyVaultAppSecret;
+
+        [Parameter("AzureKeyVaultCertificateName")] 
+        readonly string? AzureKeyVaultCertificateName;
+
+        [Parameter("SigningCertificatePath")]
```

Comment:
Good point, fixed.

I don't think these are really specified anywhere because the defaults are used but you're right, I should keep them the same as the previous parameter names.